<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnView</class>
 <widget class="QMainWindow" name="ConnView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>212</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>232</width>
    <height>212</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>232</width>
    <height>212</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Connect</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QRadioButton" name="radiobtn_IP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect by IP and Port</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelIP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input IP:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputIP">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelPort">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputPort">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radiobtn_auto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>161</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect automatically</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radiobtn_local">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>171</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Search on current host</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radiobtn_LAN">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>111</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Search on LAN</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>232</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
